docs/app/services/plunker/plunker.service.ts

```diff
@@ -27,18 +30,34 @@ export class PlunkerService {
 
     private initForm(title: string, plunk: IPlunk): HTMLFormElement {
 
+        let modules = (plunk.modules && plunk.modules.imports) ? plunk.modules.imports: '';
+        let library = (plunk.modules && plunk.modules.library) ? plunk.modules.library: '';
+        let imports = (modules && library) ? 'import { ' + modules + ' } from \'' + library + '\';': '';
+
+        let map = ''
+        if(plunk.modules && plunk.modules.map) {
+            for(let i = 0; i<plunk.modules.map.length; i++) {
```

Comment:
Although this will have to change a bit when you make the change to the interface, this whole section can be simplified to something like this:

```typescript
plunk.modules.map.map(mapping => `'${ mapping.alias }': '${ mapping.source}'`).join(',\n\t\t');
```